audio-worklet/index.html

```diff
@@ -121,7 +121,62 @@ <h4>
               </p>
             </div>
           </div>
-        </div><br>
+        </div>
+        <br>
+        <br>
+        <h2>
+          Advanced: Design Patterns
+        </h2>
+        <div class="row">
+          <div class="column">
+            <div class="demo-item">
+              <a href="design-pattern/awn/index.html">
+              <h4>
+                AudioWorklet and WebAssembly
+              </h4></a>
+              <p>
+                A basic set up for AudioWorklet and WebAssembly.
+              </p>
+            </div>
+          </div>
+          <div class="column">
+            <div class="demo-item">
+              <a href="">
+              <h4>
+                AudioWorklet and WebAssembly with RingBuffer
+              </h4></a>
+              <p>
+                How to reconcile different buffer size between AudioWorklet and
+                WASM function.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column">
+            <div class="demo-item">
+              <a href="">
+              <h4>
+                AudioWorklet and SharedArrayBuffer
+              </h4></a>
+              <p>
+                AudioWorklet, Worker and SharedArrayBuffer living in harmony.
+              </p>
+            </div>
+          </div>
+          <div class="column">
+            <div class="demo-item">
+              <a href="">
+              <h4>
+                ScriptProcessorNode and WebAssembly
+              </h4></a>
+              <p>
+                Deprecated, but for its legacy.
```

Comment:
What? Revise this a bit somehow to explain what you mean.

Reply:
I created an example of SPN and WASM as well, but I am on the fence. We can just take it out - I don't have a strong preference.

Reply:
I think it's ok to have one example that works with both SPN and AWN (especially if it shows SPN doesn't work so well), but I don't think we need to do this for all examples.

Reply:
I am removing this section until I add the actual example.